src/main/java/com/redhat/management/approval/Identity.java

```diff
@@ -1,56 +1,58 @@
 package com.redhat.management.approval;
 
+import java.io.Serializable;
+
 /**
  * This class was automatically generated by the data modeler tool.
  */
 
-public class Identity implements java.io.Serializable {
+public class Identity implements Serializable {
 
     static final long serialVersionUID = 1L;
 
-    private java.lang.String account_number;
-    private java.lang.String type;
-    private com.redhat.management.approval.User user;
-    private com.redhat.management.approval.Internal internal;
+    private String account_number;
```

Comment:
please add comment why here the field name must be `account_number` rather than `accountNumber`

Reply:
The comment can be something like 
// This class models an identity object passed in as a map.
// The attribute names must exactly match the keys in the map for object mapping.